### PR TITLE
Feature/Add CC-BY-NC-ND and CC-BY-SA4-0 and Remove "Other" License Option on Preprint Providers [PLAT-722] 

### PR DESCRIPTION
**Pull request by pattisdr**
# Purpose
❗️ Is dependent on https://github.com/CenterForOpenScience/list-of-licenses/pull/4

## Part I
Add the following licenses to OSF license options dropdown:

CC-BY-NC-ND (requested by Earthrxiv)
CC-BY-SA 4.0 (requested by LawArXiv and Thesis Commons user)

screen shot 2018-03-21 at 7 08 38 pm

## Part II
We shouldn't allow a preprint provider to be configured w/the "other" as an license option. It also appears in the options for default license. Remove 'other' license option type from the preprint configuration list in the admin app.

### Before
screen shot 2018-04-02 at 5 36 43 pm

### After
screen shot 2018-04-02 at 5 38 49 pm

# Ticket
https://openscience.atlassian.net/browse/PLAT-722

### CODE REVIEW

**Comment by caseyrollins** [comment]
@pattisdr, could you please add two screenshots to the PR description in lieu of product demo:
1. The new licenses as options in the create preprint provider form in the admin app
2. The new licenses as options in the dropdown on the OSF project overview page (both should be under "content" -- see ticket comments for additional info)

Thank you!

**Comment by pattisdr** [comment]
Done @caseyrollins . Was a little time consuming to set this up without merging the dependency https://github.com/CenterForOpenScience/list-of-licenses/pull/4, but worth it because I caught a couple of mistakes!

**Comment by caseyrollins** [comment]
Thank you so much, @pattisdr!

**Comment by pattisdr** [comment]
failing test unrelated

File: admin/base/utils.py [hunk 1]
```diff
@@ -97,8 +97,12 @@ def get_subject_rules(subjects_selected):
 
 
 def get_nodelicense_choices():
-    return NodeLicense.objects.values_list('id', 'name')
+    return NodeLicense.objects.exclude(license_id='OTHER').values_list('id', 'name')
```
Comment (erinspace) [inline admin/base/utils.py]: Is 'OTHER' some test license?  Is there a possibility that there will be more test licenses made in the future?  Main question is weather or not 'OTHER' should be hard-coded, or if we should make that a variable at the top that can be added to? If it's a one off, no need to change this!

Comment (pattisdr) [reply admin/base/utils.py]: Hey @erinspace, thanks for the review.  My understanding, the "Other" option allows users to upload a license.txt file, but this isn't currently supported on the preprints workflow.  So I am just removing it as an option from the preprint provider portion of the admin app.  Other is still an option on projects.

screen shot 2018-04-04 at 10 00 46 am

**Comment by sloria** [comment]
@pattisdr I've published list-of-licenses 1.1.0 with your PR. Can you bump the version in package.json and add a new migration that adds those 2 new licenses to the database? I believe there's an `ensure_licenses` function to do that.

**Pull request merged**